Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
-ruby '~> 3.0.3'
+ruby "~> #{File.read('.ruby-version').strip}"
```

Comment:
Should we drop the `~>` to match the specificity in `.ruby-version`?

Reply:
Good question. I assumed we wanted to allow newer patch versions since it also provides flexibility in the deployed environments, but I don't really know much about those environments. 

I would be on the side of keeping it, so we don't force people on later patches to revert back to 3.0.3.

Reply:
I don't feel strongly on it, so that seems fine to me!